### ISSUE TITLE
Left sub-nav color is too light

### ISSUE THREAD

**Issue by rsadres** [cockroachdb/docs#9435]
![Screen Shot 2021-01-21 at 11 31 03 AM](https://user-images.githubusercontent.com/10146234/105382729-62991900-5bde-11eb-9ee3-5312bcab23aa.png)

The color of the sub-nav text is too light. The color combined with the size makes it fail accessibility - as you can see in the screenshot. 

Also, the open/close carats seem backward to me. I can't tell if the section is open or not unless I click to make sure.

**Comment by rsadres** [comment]
![Screen Shot 2021-01-21 at 11 54 34 AM](https://user-images.githubusercontent.com/10146234/105383785-7d1fc200-5bdf-11eb-9e22-4497da22a250.png)
![Screen Shot 2021-01-21 at 11 54 30 AM](https://user-images.githubusercontent.com/10146234/105383787-7db85880-5bdf-11eb-9f89-d0a76a418131.png)
Step numbers and TOC also seem too light.

**Comment by luciaaaaaaa** [comment]
Closing in favor of [Sidebar navigation color and font-size #9424](https://github.com/cockroachdb/docs/issues/9424)
